lib/ansible/module_utils/basic.py

```diff
@@ -80,7 +80,7 @@
 import platform
 import errno
 import datetime
-from itertools import repeat, chain
+from itertools import repeat, chain, imap
```

Comment:
There is no `imap` in `itertools` in python 3:

http://stackoverflow.com/questions/30271712/i-cant-find-imap-in-itertools-in-python